docs/compression/README.md

```diff
@@ -0,0 +1,75 @@
+# Compression Guide [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io)
+
+The compression feature in Minio benifits the tenants by efficient usage of the disk space. Minio, By default compresses all the compressible objects.
```

Comment:
You can start with something like this 

Minio server has compression enabled by default. This means files uploaded to the server are compressed before being written to disk(s). This enables efficient disk space usage. 

Minio compression is based on [Snappy](https://github.com/golang/snappy) library. This allows  

- <Benefits of Snappy over others>